pkg/compiler/codegen.go

```diff
@@ -261,8 +261,10 @@ func (c *codegen) convertFuncDecl(file ast.Node, decl *ast.FuncDecl) {
 
 	ast.Walk(c, decl.Body)
 
-	// If this function returns the void (no return stmt) we will cleanup its junk on the stack.
-	if !hasReturnStmt(decl) {
+	// If we have reached the end of the function without encountering `return` statement,
+	// we should clean alt.stack manually.
+	// This can be the case with void and named-return functions. For now we support only former case.
```

Comment:
Do we have an issue for functions with return variables?

Reply:
Not anymore :)

Reply:
Well, then this comment is a bit outdated now.